examples/nemo/scripts/process_nemo.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python3
```

Comment:
As agreed, I'm going to let this and kernels_trans.py on to master without any changes to make it easier to keep them up to date. For the record though, I think this particular script would be useful to re-work into as a generic batch_process script. I've often had to write something to run over all lfric files, for example. I think the only nemo-specific thing here is the list of excluded files which could be stored in a config file, or similar.

Reply:
Agreed. I've created #1142.